tutorials/commands/flux-proxy-command.rst

```diff
@@ -0,0 +1,194 @@
+.. _flux-proxy-command:
+
+=================
+Flux Proxy Basics
+=================
+
+With Flux, it is very common to create a Flux :ref:`subinstances<subinstance>`.  However, it can be confusing how to interact with those subinstances.
```

Comment:
So this is saying when I launch a flux job that controls some set of subinstances, I would use flux proxy to communicate with them? Or could I have flux controlling multiple clusters, and then flux proxy would give me access to a cluster? What is a subinstance (or what can it be) in concrete terms? Why would I want to communicate with it? E.g., what we might have here is "you might want to do this if X or Y or Z" is what you are doing.

Reply:
Maybe a better intro here would be to talk about traditional batch jobs running a batch script, and how in Flux those "batch jobs" are actually running a full instance of Flux, and the `flux proxy` command can be used to execute processes, including an interactive shell, as if they were connected directly to the instance of Flux running your batch workload.

Also, `flux proxy` is not restricted to running against subinstances. You can also use it to connect to a system instance running on another cluster, a test instance running locally, or as you show below, a Flux instance running under a Slurm, LSF, or other resource manager job.

Reply:
> Maybe a better intro here would be to talk about traditional batch jobs running a batch script, and how in Flux those "batch jobs" are actually running a full instance of Flux, and the flux proxy command can be used to execute processes, including an interactive shell, as if they were connected directly to the instance of Flux running your batch workload.

The way I imagined it was that we'd eventually have this discussed more in a `flux mini batch` tutorial.  Although I can probably go into it a little more than I have so the reader isn't quite as clueless.